robotis-op2/menu.md

```diff
@@ -15,19 +15,21 @@
     - [Walk](walk.md)
     - [Soccer](soccer.md)
 - [Robot Window](robot-window.md)
-    - [Accelerometers](accelerometers.md)
-    - [Cameras](cameras.md)
-    - [Gyro](gyro.md)
-    - [Servos](servos.md)
-- [Cross-Compilation](cross-compilation.md)
+    - [Accelerometer](accelerometer.md)
+    - [Gyroscope](gyroscope.md)
```

Comment:
The title in gyroscope.md is still Gyro.
Note that Gyro is the Webots device name.